### ISSUE TITLE
Support for Swiftmailer 6.0

### ISSUE THREAD

**Issue by Taluu** [mailjet/MailjetSwiftMailer#9]
Swiftmailer 6.0 was released something like 6 months ago, and IRC, should not have too many changes impacting a simple Transport.

Any support planned for 6.0 ? :}

**Comment by Nightbr** [comment]
We support Smailer 6: 

* https://github.com/mailjet/MailjetSwiftMailer/issues/4
* https://github.com/mailjet/laravel-mailjet/issues/8

**Comment by Taluu** [comment]
Nevermind, it seems there is some incompabilities (e.g removal of `Swift_Mime_Message`).

**Comment by Taluu** [comment]
Indeed, any plans on releasing a version ?

**Comment by Nightbr** [comment]
yes, very soon I hope... I would like to create a release with BC but tag it major and update all projects (mailjetBundle + laravel-mailjet) with it around the beginning of 2018. Waiting for new contacts from mailjet team

**Comment by latanasov** [comment]
cc @Zhivko-Mailjet  @eboisgon @kachar

**Comment by jGielen** [comment]
@Nightbr would it not be more logic to require "symfony/swiftmailer-bundle" instead of "swiftmailer/swiftmailer"?

**Comment by Nightbr** [comment]
@jGielen you can't require `symfony/swiftmailer-bundle` because it only provides Symfony integration for SwiftMailer. This is a swiftmailer transport for Mailjet, it is used in `mailjet/MailjetBundle` and `mailjet/laravel-mailjet` packages. Add to this, you can use this on raw php project with SwiftMailer.

So no, it's not logic to require `symfony/swiftmailer-bundle` an it will not resolve the problem of compatibility btw...
